### ISSUE TITLE
Installation plugins is failed when OS is alpine

### ISSUE THREAD

**Issue by Ra1nmaker** [IBM-Cloud/ibm-cloud-cli-release#29]
Hi,
I want to install the following clis and plugins into alpine linux to control IBM Container service automatically:
* bx cli
* container-registry plugin
* container-service plugin, and 
* kubectl cli 

Installation bx cli , kubectl cli, and container-registry plugin was successful, but container-service plugin was failed.

Error I faced is as follows:
```
# This output is for container-registry
Looking up 'container-registry' from repository 'Bluemix'...
Plug-in 'container-registry 0.1.304' found in repository 'Bluemix'
Attempting to download the binary file...
 29.30 MiB / 29.30 MiB  100.00% 19s2ss
30725403 bytes downloaded
Installing binary...
OK
Plug-in 'container-registry 0.1.304' was successfully installed into /root/.bluemix/plugins/container-registry. Use 'bx plugin show container-registry' to show its details.

# This output is for container-service
Looking up 'container-service' from repository 'Bluemix'...
Plug-in 'container-service 0.1.481' found in repository 'Bluemix'
Attempting to download the binary file...
 20.21 MiB / 20.21 MiB  100.00% 14ssss
21188481 bytes downloaded
Installing binary...
FAILED
Unable to obtain plug-in's metadata. Error: fork/exec /tmp/BluemixFileDownload104351476/container-service-linux-amd64-0.1.481: no such file or directory
```

Is Alpine Linux not supported?

**Comment by boyang9527** [comment]
@jsloyer can you have someone from your team looking at this issue of the container service plugin?

**Comment by bhpratt** [comment]
This work-around came from another team member. Not a permanent solution, since it requires manually maintaining the version, but it does work.
```
# version of container-service plugin to install
ENV CS_VERSION 0.1.488
# we need to grab the 386 arch as the amd64 version doesn't install correctly on alpine
# NOTE(cjschaef): also, the version number may not be appended to the binary name in future, so watch out
ADD http://public.dhe.ibm.com/cloud/bluemix/cli/bluemix-plugins/container-service/${CS_VERSION}/container-service-linux-386-${CS_VERSION} /tmp
RUN bx plugin install /tmp/container-service-linux-386-${CS_VERSION} && bx plugin list && \
    rm -f /tmp/container-service-linux-386-${CS_VERSION}
```

Currently Alpine only works with the x86 architecture. @boyang9527 would it be possible to add a flag to the plugin that allows the user to specify x86 architecture when installing/updating the plugin?

Note, you can use other versions, like Ubuntu, for your Dockerfile w/o specifying x86 arch.

**Comment by boyang9527** [comment]
it only happens on container service. The bluemix cli and container-registry plugin  64bit version works fine... so I suspect there is some issue when container service team is compiling the plugin....

**Comment by jsloyer** [comment]
@Ra1nmaker are you still having this issue?

**Comment by bhpratt** [comment]
This is fixed with plugin version 0.1.504.

Thanks. @boyang9527 can you close this issue? It doesn't appear that I have permissions to do so.
